### PR TITLE
[MS] polish selection item modal

### LINKED ISSUE #5848: [🦋 Final touch] Polish FolderSelectionModal to match mockups
See #5814

### PR DESCRIPTION
**Pull request by fabienSvtr**
## Describe your changes

Update the style to mockup.
image

⚠️ Need for change:
- `Back`/`Next` button (following issue #6056)

## Checklist

Before you submit this pull request, please make sure to:

- [ ] Keep changes in the pull request as small as possible
   
- [ ] Ensure the commit history is sanitized
   
- [ ] Give a meaningful title to your PR
- [ ] Describe your changes
   
- [ ] Link any related issue in the description
   
- [ ] Link any dependent pull request in the description

closes #5848

### CODE REVIEW

File: client/src/components/core/ms-modal/FolderSelectionModal.vue [hunk 1]
```diff
@@ -3,73 +3,87 @@
...
-      @change="onPathChange"
-    />
-    <div>
-      <ion-list>
-        <ion-list-header
-          class="folder-list-header"
-          lines="full"
+    <div class="navigation">
+      <ion-buttons>
+        <ion-button
```
Comment (Max-7) [inline client/src/components/core/ms-modal/FolderSelectionModal.vue:20]: Those two buttons are completly wrong right now. Either there're getting fixed in this PR, in which case the PR should be a draft, or their logic should be removed and the buttons should be hidden.

File: client/src/components/core/ms-modal/FolderSelectionModal.vue [hunk 2]
```diff
@@ -3,73 +3,87 @@
...
 import { FolderSelectionOptions, MsModalResult } from '@/components/core/ms-modal/types';
 import HeaderBreadcrumbs, { RouterPathNode } from '@/components/header/HeaderBreadcrumbs.vue';
-import { EntryStat, EntryStatFolder, Path, entryStat } from '@/parsec';
-import { IonIcon, IonItem, IonLabel, IonList, IonListHeader, modalController } from '@ionic/vue';
-import { document, folder, home } from 'ionicons/icons';
+import { EntryStat, EntryStatFolder, Path, entryStat, getWorkspaceName } from '@/parsec';
+import { IonButton, IonButtons, IonIcon, IonItem, IonLabel, IonList, modalController } from '@ionic/vue';
+import { chevronBack, chevronForward, document, folder } from 'ionicons/icons';
 import { Ref, onMounted, ref } from 'vue';
+import { useRouter } from 'vue-router';
```
Comment (Max-7) [inline client/src/components/core/ms-modal/FolderSelectionModal.vue:79]: The router should not be appear in this file.

File: client/src/components/core/ms-modal/FolderSelectionModal.vue [hunk 3]
```diff
@@ -3,73 +3,87 @@
...
 
+const router = useRouter();
 const props = defineProps<FolderSelectionOptions>();
 const selectedPath = ref(props.startingPath);
 const headerPath: Ref<RouterPathNode[]> = ref([]);
 const currentEntries: Ref<EntryStat[]> = ref([]);
+const backButtonDisabled = ref(true);
```
Comment (Max-7) [inline client/src/components/core/ms-modal/FolderSelectionModal.vue:86]: Useless logic.

File: client/src/components/core/ms-modal/FolderSelectionModal.vue [hunk 4]
```diff
@@ -91,12 +105,17 @@ async function update(): Promise<void> {
   }
   currentEntries.value.sort((item1, item2) => Number(item1.isFile()) - Number(item2.isFile()));
 
+  let workspaceName = '';
+  const workspaceResult = await getWorkspaceName(props.workspaceId);
```
Comment (Max-7) [inline client/src/components/core/ms-modal/FolderSelectionModal.vue:109]: Should be on mount.

File: client/src/components/core/ms-modal/FolderSelectionModal.vue [hunk 5]
```diff
@@ -135,34 +153,42 @@ async function confirm(): Promise<boolean> {
 async function cancel(): Promise<boolean> {
   return modalController.dismiss(null, MsModalResult.Cancel);
 }
+
+function goBack(): void {
```
Comment (Max-7) [inline client/src/components/core/ms-modal/FolderSelectionModal.vue:157]: Remove.

File: client/src/components/core/ms-modal/FolderSelectionModal.vue [hunk 6]
```diff
@@ -135,34 +153,42 @@ async function confirm(): Promise<boolean> {
...
+  backButtonDisabled.value = false;
+  router.back();
+}
+
+function goForward(): void {
```
Comment (Max-7) [inline client/src/components/core/ms-modal/FolderSelectionModal.vue:162]: Remove.

File: client/src/components/core/ms-modal/types.ts [hunk 7]
```diff
@@ -13,6 +13,7 @@ export interface FolderSelectionOptions {
   title: string;
   subtitle?: string;
   startingPath: FsPath;
+  workspaceId: string;
```
Comment (Max-7) [inline client/src/components/core/ms-modal/types.ts:16]: Should be a `WorkspaceID`.

File: client/tests/e2e/specs/test_folders_page.ts [hunk 8]
```diff
@@ -206,9 +206,6 @@ describe('Check folders page', () => {
     cy.get('.folder-container').find('.file-list-item').eq(1).find('ion-checkbox').invoke('show').click();
     cy.get('#button-moveto').contains('Move to').click();
     cy.get('.folder-selection-modal').find('.ms-modal-header__title').contains('Move one item');
-    cy.get('.folder-selection-modal')
```
Comment (Max-7) [inline client/tests/e2e/specs/test_folders_page.ts:209]: This code should be replaced by a code that check that the current path is correct. Same for every deletion in this file.

**Review by Max-7 — CHANGES_REQUESTED** [review]
The header breadcrumb appears quite big. Maybe we could set the number of items before/after collapse to 1 in this case instead of 2 by adding props to HeaderBreadcrumbs https://github.com/Scille/parsec-cloud/blob/583d340a827d8e5de672e545aa342e40a11941fa/client/src/components/header/HeaderBreadcrumbs.vue#L8

**Comment by fabienSvtr** [comment]
> The header breadcrumb appears quite big. Maybe we could set the number of items before/after collapse to 1 in this case instead of 2 by adding props to HeaderBreadcrumbs
> 
> https://github.com/Scille/parsec-cloud/blob/583d340a827d8e5de672e545aa342e40a11941fa/client/src/components/header/HeaderBreadcrumbs.vue#L8

Yes, I added a `props` to set the value depending the case (`FolderPage` or `Selection Modal`) .

File: client/src/components/core/ms-modal/FolderSelectionModal.vue (hunk 1 above)
Comment (fabienSvtr) [reply client/src/components/core/ms-modal/FolderSelectionModal.vue:20]: As mention in the description of the PR, this issue will be fixed in https://github.com/Scille/parsec-cloud/issues/6056

**Review by Max-7 — APPROVED** [review]

**Pull request merged**